backend/src/test/java/com/google/sps/FindMatchQueryTest.java

```diff
@@ -112,6 +126,9 @@ public void compatibleTimeAndDuration() {
             TIME_1400ET,
             TIME_1600ET,
             DURATION_15_MINUTES,
+            ROLE_SOFTWARE_ENGINEER,
+            PRODUCT_AREA_ADS,
+            MATCH_PREFERENCE_SIMILAR,
             MATCHID_DEFAULT,
             MATCHSTATUS_UNMATCHED,
             TIMESTAMP_DEFAULT);
```

Comment:
I noticed that a lot of logic is repeated in created participates, do you think it would be good to extract some of it to a helper function ?

Reply:
What kind of helper function? I think it helps to see all of the fields in each participant so that debugging each case is easier.

Reply:
I do agree that as it stands right now it is very  blatant as to what the fields are,but, I believe a create Participant function just for this test suite would be fine ?.